### ISSUE TITLE
[docs] Improve documentation for tauri

### ISSUE THREAD

**Issue by dclong** [tauri-apps/tauri#1697]
The project tauri is very interesting. However, the documentation is a little hard to follow. More examples/showcases would also be extremely helpful.

**Comment by lucasfernog** [comment]
We're currently improving the examples and documentation, see #1685 #1687 #1690 #1692 #1694.
We would love to hear your thoughts and suggestions, please join the #documentation channel on Discord. Also, we're open to contributions :)
